adhocracy4/polls/signals.py

```diff
@@ -0,0 +1,3 @@
+from django.dispatch import Signal
+
+poll_voted = Signal()
```

Comment:
Can you add here the arguments that are needed for the signal.

Reply:
added a docstring